src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import javax.persistence.*;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "order")
+public class Order {
+    enum OrderStatus {
+        UNPAID,
+        PAID,
+        CANCELED
+    }
+
```

Comment:
enum mam w bibliotece tak zrobiony:
@Column
    @NotNull
    @Enumerated(EnumType.STRING)
    private Status status;
i enum jest utworzony w pakiecie domain tak jak się tworzy klasę, czy interface, tylko trzeba ustawic, że to enum.

Reply:
Ja wolę enuma w klasie, szczególnie takiego małego, bo od razu widać co i jak. Oba sposoby są okej.